ecs-cli/modules/clients/aws/ec2/client.go

```diff
@@ -85,3 +87,29 @@ func (c *ec2Client) DescribeNetworkInterfaces(networkInterfaceIDs []*string) ([]
 	}
 	return response.NetworkInterfaces, nil
 }
+
+func (c *ec2Client) DescribeInstanceTypeOfferings(location string) ([]string, error) {
```

Comment:
Nit: I see in that location can be region or AZ. Since we only use region as specified in the input struct, would it help to name this as "region" instead of "location"?

Reply:
Sure! Addressed.